multinet/uploaders/d3_json.py

```diff
@@ -107,28 +108,20 @@ def upload(workspace: str, graph: str) -> Any:
         del link["target"]
 
     # Create or retrieve the workspace
-    if space.has_collection(node_table_name):
-        nodes_coll = space.collection(node_table_name)
+    if loaded_workspace.has_table(node_table_name):
+        node_table = loaded_workspace.table(node_table_name)
     else:
-        nodes_coll = space.create_collection(node_table_name, edge=False)
+        node_table = loaded_workspace.create_table(node_table_name, edge=False)
 
-    if space.has_collection(edge_table_name):
-        links_coll = space.collection(edge_table_name)
+    if loaded_workspace.has_table(edge_table_name):
+        edge_table = loaded_workspace.table(edge_table_name)
     else:
-        links_coll = space.create_collection(edge_table_name, edge=True)
+        edge_table = loaded_workspace.create_table(edge_table_name, edge=True)
```

Comment:
This seems like it could be buggy. Should we reject the upload if there is already a table with a name that we need?

Reply:
If that's a bug, it's independent of this PR. Could you file an issue please?

Reply:
Opened #458.